src/xstate/actors/audioPlayer/audioPlayerMachine.ts

```diff
@@ -419,10 +409,10 @@ export const audioPlayerMachine =
                     },
                     PLAY_AYAH: [
                       {
-                        actions: ['setNewSurahAndAyahNumbers'],
+                        actions: ['setSurahAndAyahNumbers'],
```

Comment:
everywhere we added `actions: ['setSurahAndAyahNumbers'],`, we should also add `exitRadio` and `stopRepeatActor` actions which we were triggering when `CONFIRM_PLAY_MISMATCHED_SURAH ` is fired. This is necessary to exit repeat mode/radio mode when the user plays another Surah.
<img width="374" alt="Screen Shot 2022-09-24 at 7 35 42 PM" src="https://user-images.githubusercontent.com/15169499/192098286-a4a202ba-84fa-4cae-a518-f62362f848d4.png">